react/src/components/ServiceLauncherModal.tsx

```diff
@@ -641,6 +739,59 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
                     </Form.Item>
                   )
                 )}
+                <Flex
+                  direction="row"
+                  gap={'xxs'}
+                  align="stretch"
+                  justify="between"
+                >
+                  <Form.Item
+                    name={'modelMountDestination'}
+                    label={t('modelService.ModelMountDestination')}
+                    style={{ width: '50%' }}
+                    labelCol={{ style: { width: 400 } }}
+                  >
+                    <Input
+                      allowClear
+                      placeholder={'/models'}
+                      disabled={!!endpoint}
+                    />
+                  </Form.Item>
+                  <MinusOutlined
+                    style={{
+                      fontSize: token.fontSizeXL,
+                      color: token.colorTextDisabled,
+                    }}
+                    rotate={290}
+                  />
+                  <Form.Item
+                    name={'modelDefinitionPath'}
+                    label={t('modelService.ModelDefinitionPath')}
+                    style={{ width: '50%' }}
+                    labelCol={{ style: { width: 300 } }}
+                  >
+                    <Input
+                      allowClear
+                      placeholder={
+                        endpoint?.model_definition_path
+                          ? endpoint?.model_definition_path
+                          : 'model-definition.yaml'
+                      }
+                    />
+                  </Form.Item>
+                </Flex>
+                <VFolderTableFormItem
+                  rowKey={'id'}
+                  label={t('modelService.AdditionalMounts')}
+                  filter={(vf) =>
+                    vf.name !== formRef.current?.getFieldValue('vFolderName') &&
+                    vf.status === 'ready' &&
+                    vf.usage_mode !== 'model'
+                  }
+                  tableProps={{
+                    size: 'small',
+                  }}
+                />
```

Comment:
If you do not wrap the `<Form.Item noStyle shouldUpdate={...}>`, this part will not re-render when `vFolderName` is changed. Please check [my previous comment](https://github.com/lablup/backend.ai-webui/pull/2437#discussion_r1619562472) again.